### ISSUE TITLE
Option to make the banner colour exactly the same as the navbar colour (dark-www)

### ISSUE THREAD

**Issue by Secret-chest** [ScratchAddons/ScratchAddons#5971]
### Why this would be helpful

_fancy issue forms_

### My suggestion

_fancy issue forms_

### Possible alternatives

We could also remove the _fancy issue forms_ instead.

### Additional context

_fancy issue forms_

**Comment by WorldLanguages** [comment]
The issue forms are fancy but I don't know what banner you're referring to.

**Comment by Secret-chest** [comment]
The banners like the ones for My Stuff, Discuss Scratch etc.

**Comment by WorldLanguages** [comment]
I think this was discussed somewhere but I can't find it right now.

Btw, keep in mind that My Stuff only has a banner if "Scratch 2.0→3.0" is enabled.

**Comment by Secret-chest** [comment]
Yes, but the one in the Explore page is always orange. I'd also like to change it, but that's a separate issue.
